cargo-shuttle/src/args.rs

```diff
@@ -131,14 +131,18 @@ pub enum Command {
     Login(LoginArgs),
     /// Log out of the Shuttle platform
     Logout(LogoutArgs),
-    /// Generate shell completions
+    /// Generate shell completions and manpage
     Generate {
         /// Which shell
         #[arg(short, long, env, default_value_t = Shell::Bash)]
         shell: Shell,
         /// Output to a file (stdout by default)
         #[arg(short, long, env)]
         output: Option<PathBuf>,
+        /// Generate manpage in the directory specified by the `OUT_DIR`
+        /// environment variable (current working directory by default)
+        #[arg(short, long)]
+        manpage: bool,
```

Comment:
How about we don't use `OUT_DIR` and print it out to `stdout` instead? (similar to shell completions)

Reply:
Wait a minute... `--shell` supports out dir... I gotta say though, it feels cleaner to only allow stdout, and let the user pipe to a file.

Reply:
Agree. File I/O can be expensive, and imo the logic inside `complete` would seem simpler if we were to only support stdout for shell completions as well.